doc/source/conf.py

```diff
@@ -146,6 +146,14 @@ def setup(app):
     app.add_config_value('python_version_major', str(sys.version_info.major), 'env')
     app.add_lexer('NumPyC', NumPyLexer)
 
+# While these objects do have type `module`, the names are aliases for modules
+# elsewhere. Sphinx does not support referring to modules by an aliases name,
+# so we make the alias look like a "real" module for it.
+# If we deemed it desirable, we could in future make these real modules, which
+# would make `from numpy.char import split` work.
+sys.modules['numpy.char'] = numpy.char
```

Comment:
This should fix some more warnings
```suggestion
sys.modules['numpy.rec'] = numpy.rec
```

Reply:
I tried, but it seems to make no effect.